### PR TITLE
Feature amazon s3 #25

### PR DESCRIPTION
**Pull request by stefanleoussis**
### What this PR does:
- Enables users to upload a profile Image

### Screenshots / Videos:
Screen Shot 2021-06-07 at 12 23 47 AM
Screen Shot 2021-06-07 at 12 23 57 AM
Screen Shot 2021-06-07 at 12 13 38 AM
Screen Shot 2021-06-07 at 12 15 07 AM

### Any information needed to test this feature:
  1. Login/Create an account
  2. Head to the options menu next to the profile photo, select profile Image
  3. Drag or select a profile Image, notice that dropping anything other than an Image will produce an error
  4. Hit submit and Watch app state update automatically with new Profile Photo

### CODE REVIEW

**Comment by stefanleoussis** [comment]
Merged multiple times because I was resolving conflicts on Github that didn't update when I pulled so I had to resolve it twice.

**Review by dproc96 — APPROVED** [review]
Looks good to go

File: server/controllers/auth.js [hunk 1]
```diff
@@ -59,18 +61,19 @@ exports.loginUser = asyncHandler(async (req, res, next) => {
 
     res.cookie("token", token, {
       httpOnly: true,
-      maxAge: secondsInWeek * 1000
+      maxAge: secondsInWeek * 1000,
     });
-
+    console.log(user);
```
Comment (dproc96) [inline server/controllers/auth.js:66]:
```suggestion
```

File: server/controllers/auth.js [hunk 2]
```diff
@@ -88,15 +91,17 @@ exports.loadUser = asyncHandler(async (req, res, next) => {
     res.status(401);
     throw new Error("Not authorized");
   }
+  console.log(user + "This is User");
```
Comment (dproc96) [inline server/controllers/auth.js:94]:
```suggestion
```

Comment (stefanleoussis) [reply server/controllers/auth.js:93]: done

**Pull request merged**